ui/app/models/mfa-login-enforcement.js

```diff
@@ -0,0 +1,28 @@
+import Model, { attr, hasMany } from '@ember-data/model';
+import { expandAttributeMeta } from 'vault/utils/field-to-attrs';
+
+export default class mfaLoginEnforcementModel extends Model {
+  @attr('string', {
+    label: 'Enforcement name',
+    subText:
+      'The name for this enforcement. Giving it a name means that you can refer to it again later. This name will not be editable later.',
```

Comment:
More of curious question: are there any characters that the enforcement cannot have e.g. commas, spaces, *, etc. If so maybe later we can add validation.

Reply:
That's a good question. If there were I would expect the API to return an error so that's something we should test once the form is built and then we can add validation on our end.